extensions/amp-analytics/0.1/crypto-impl.js

```diff
@@ -93,6 +104,20 @@ export class Crypto {
       return result;
     });
   }
+
+  /**
+   * Loads Crypto polyfill library.
+   * @returns {!Promise}
```

Comment:
This should be `!Promise<????>` since you are using response of this promise everywhere.